docs/pages/components/list.md

```diff
@@ -111,6 +111,55 @@ If only a subset of the list items are navigable you should indicate those by di
 
 {% include display-component.html component=list-with-links-navigation %}
 
+## Standard List with Action List Item
+
+{% capture standard-list-action %}
+<ul class="fd-list" role="list">
+  <li role="listitem" tabindex="0" class="fd-list__item fd-list__item--action">
+      <span class="fd-list__title">Forward</span>
```

Comment:
It seems as if these should be `button`s instead of `span`s if the idea is they will trigger an action. Reading these with VoiceOver, they're currently indistinguishable from the plain list items.